Content.Server/Atmos/Components/BreathToolComponent.cs

```diff
@@ -16,5 +16,12 @@ public sealed partial class BreathToolComponent : Component
         public SlotFlags AllowedSlots = SlotFlags.MASK | SlotFlags.HEAD;
         public bool IsFunctional;
         public EntityUid? ConnectedInternalsEntity;
+
+        /// <summary>
+        /// Tool will automatically turn on internals when it is next equipped
+        /// This feature will turn itself off after the first time it's used
+        /// </summary>
+        [DataField("automaticActivation")]
+        public bool AutomaticActivation = false;
     }
```

Comment:
Datafield strings are obsolete.

```suggestion

        /// <summary>
        /// Tool will automatically turn on internals when it is next equipped
        /// This feature will turn itself off after the first time it's used
        /// </summary>
        [DataField]
        public bool AutomaticActivation = false;
    }
```